### ISSUE TITLE
fonts not found when loading off cdn due to url getting mangled

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetML#186]
I just published `@doenet/standalone`  as a separate npm package so that we can serve it off the jsdelivr cdn. I just packaged the old 0.6 version (the one currently on doenet.org) and published it so one can get it via:

```
<script type="module" src="https://cdn.jsdelivr.net/npm/@doenet/standalone@0.6.1/dist/doenet-standalone.js"></script>        <script src="https://www.youtube.com/iframe_api"></script>
<link href="https://cdn.jsdelivr.net/npm/@doenet/standalone@0.6.1/dist/style.css" rel="stylesheet" type="text/css" />
```

However, when I load a page, I get the error that it cannot load
```https://cdn.jsdelivr.net/npm/@doenet/standalone@0.6.1/fonts/files/open-sans-v18-latin-regular.woff2```
The location of the file is `/dist/fonts/`, but it lost the `/dist` directory, so it cannot find the font.

**Comment by siefkenj** [comment]
Yes, the path that the fonts are retrieved from is rewritten by one of the compilation processes...This is not an ideal situation :-/ and one reason why it would be good to have a packaging script that creates a new `package.json` just for the distribution. If you create that new `package.json`, then you can have all the files in the root of the package directory and there shouldn't be a pesky `dist/` hiding them.
